Version 1.3.1 (version bump)

lukecarbis opened the pull request.

lukecarbis committed:
Version bump

kienstra commented on README.md:
```diff
@@ -3,7 +3,7 @@
 Tags: Gutenberg, Blocks
 Requires at least: 5.0
-Tested up to: 5.1.1
+Tested up to: 5.2.1
```
This might be able to be simply `5.2`. That's how it is [in the AMP plugin](https://github.com/ampproject/amp-wp/blob/a042ea26011145a6ff790dc0b52697c0e813e5b4/readme.txt#L5) (though that `develop` branch isn't deployed to wp.org).

lukecarbis replied:
Might as well put the latest version, though, right?

kienstra replied:
I think it might help to make this `5.2`, so we don't have to update it for WP `5.2.2`.

It looks like WordPress.org displays the latest point version as the "Tested up to" version, if there's only a 2-digit version in `readme.txt`, like `5.2`.

For example, the AMP plugin's [Tested up to version](https://plugins.trac.wordpress.org/browser/amp/trunk/readme.txt#L5) is `5.2`. But WordPress.org displays it as `5.2.1`:

<img width="1011" alt="tested" src="https://user-images.githubusercontent.com/4063887/58148791-c27b3780-7c25-11e9-9842-c3f50634e379.png">

kienstra reviewed (dismissed):
**Looks Good**

Hi @lukecarbis,
This looks good, though I haven't done any additional testing for this PR.

kienstra commented on CHANGELOG.md:
```diff
@@ -1,5 +1,11 @@
+
+* New: Support for Gutenberg's built-in Additional CSS Class in your block template, by using the field `className`. [Read more](https://github.com/getblocklab/block-lab/wiki/7.-FAQ)
+* New: The Textarea field now has an option to automatically add paragraph tags and line-breaks
+* Fix: Bug effecting blocks containing Pro fields when there's no active Pro license
```
Maybe `effecting` should be `affecting`.

kienstra commented:
https://github.com/ampproject/amp-wp/commit/40fa940cd3c16a0b2e43ae2251c18cc381cbede7#diff-354f30a63fb0907d4ad57269548329e3 might help with the failed Travis build.

RobStino reviewed (dismissed):
Created a fairly comprehensive block and all looked good.

lukecarbis committed:
Fix spelling in changelog

kienstra commented:
Hi @lukecarbis, I'll work on this soon.

kienstra committed:
Test WP 5.1 with PHP 5.4, not WP 5.2

I think WP 5.2 doesn't support 5.4 anymore,
so see if this fixes the failed build.

kienstra reviewed (approved):
**Approved**

Hi @lukecarbis,
This is ready to merge if you're happy with it.

lukecarbis commented:
Oh, damn. Forgot to change the tested up to tag. We'll do it next time. Sorry @kienstra !

kienstra commented:
No problem, it wasn't bad as is.
